app/models/interaction.rb

```diff
@@ -2,7 +2,6 @@ class Interaction < InteractionsRecord
   scope :created_between, lambda { |start_date, end_date|
                             where(created_at: start_date.beginning_of_day..end_date.end_of_day)
                           }
-  require "csv"
```

Comment:
I'm not sure why you removed this line (also in other files)? This is not a RuboCop violation. Without this line, this class will throw an error if CSV is not magically loaded before loading this class and we don't want to risk that, right?

On another point: it'd be better to place this at the top of the file to be better visible since that's the usual place where we put the `require` statements.

Reply:
> "if `CSV` is not magically loaded before loading this class"

Turns out that's exactly what is done. But there's no magic: instead it is just the line `Bundler.require(*Rails.groups)` in `config/application.rb` which is responsible for this behavior. So indeed, removing these `require` lines as you did works just fine.

As CSV functionality is not used in many places, we might discuss if we instead want to do a `gem "csv", "~> 3.3", require: false` instead, such that we only `require` it at the respective places (at the very top of  files that use the `CSV` class). Of course, one single gem doesn't cut startup times, but in total with many gems it might make a difference.

Reply:
I think i would be in favor of reintroducing the `require` statements for csv and adding `require:false` to the corresponding line in the gemfile.

Reply:
Done in 4e1e4571f